packages/studio/src/components/IFramePortal.tsx

```diff
@@ -70,3 +55,29 @@ function useParentDocumentStyles(iframeDocument: Document | undefined) {
     }
   }, [iframeDocument]);
 }
+
+const GetCSS = (viewport: Viewport, previewRef: RefObject<HTMLDivElement>) => {
```

Comment:
Could we call this `useCSS`? With React Hooks, there's a general naming convention for the hook's name to start with `use`.